frontend/src/tests/pages/HomePage.test.js

```diff
@@ -92,53 +89,6 @@ describe("HomePage tests", () => {
       level: "G",
     });
 
-    expect(mockToast).toHaveBeenCalled();
-    //"1 Course" because only oneCourse is the result so the length of the retuened array should be 1
-    expect(mockToast).toHaveBeenCalledWith("1 Courses Retrieved");
-  });
-
-  test("calls UCSB Curriculum api correctly with 2 course response", async () => {
-    axiosMock.onGet("/api/UCSBSubjects/all").reply(200, allTheSubjects);
-    axiosMock
-      .onGet("/api/public/basicsearch")
-      .reply(200, { classes: coursesFixtures.twoCourses });
-
-    const { getByText, getByLabelText } = render(
-      <QueryClientProvider client={queryClient}>
-        <MemoryRouter>
-          <HomePage />
-        </MemoryRouter>
-      </QueryClientProvider>
-    );
-
```

Comment:
Is there a particular reason why this test is deleted?

Reply:
We no longer need the 2 course tests. We had it before because the number of courses was the only way we could confirm whether the correct subject was returned (using toasts)
now that we have the table we can directly confirm whether the correct subject was filled into the table

Reply:
ok sounds good